npm/deploy/certs/server-key.pem

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
this doesn't feel like the correct thing to do

Reply:
Yep I agree but we dont have an option here. We do not have a certificate management controller so we have to hard code these.
We could programmatically generate these but for now everything will be embedded in a Kubernetes Secret. 
Programmatically might be a better option TBH but just to move ahead we check these in since this isn't yet going into production.
Thoughts @vakalapa ?